test/e2e/operator_test.go

```diff
@@ -2642,10 +2642,17 @@ func TestAWSELBConnectionIdleTimeout(t *testing.T) {
 		t.Fatalf("expected %s=%s, found %s=%s", key, expected, key, v)
 	}
 
-	// Make sure we can resolve the route's host name.  It may take some
-	// time for the ingresscontroller's wildcard DNS record to propagate.
+	// Get the ELB's hostname via the wildcard DNS record
+	wildcardRecordName := controller.WildcardDNSRecordName(ic)
+	wildcardRecord := &iov1.DNSRecord{}
+	if err := kclient.Get(context.TODO(), wildcardRecordName, wildcardRecord); err != nil {
+		t.Fatalf("failed to get wildcard dnsrecord %s: %v", wildcardRecordName, err)
+	}
+	elbHostname := wildcardRecord.Spec.Targets[0]
```

Comment:
In a wildcard record, could there be more than one target?  How do you know which target matches the ELB?

Reply:
The DNSRecord CRD allows there to be more than one target, since a real-life DNS record can support multiple targets; however, in practice the Ingress Operator will never make a DNS record with more than 1 target based on the code here: https://github.com/openshift/cluster-ingress-operator/blob/71c424dd9d50bb3d38daecb6b3c64f6f8c9271b8/pkg/resources/dnsrecord/dns.go#L144-L170

And this assumption of 1 target is made in all of our DNS Provides, e.g.:
https://github.com/openshift/cluster-ingress-operator/blob/e8f3f45aec4ee7d1dc602bdd3ba8a3a841ddc336/pkg/dns/aws/dns.go#L503
https://github.com/openshift/cluster-ingress-operator/blob/475c9b6ec3b25d68b24d365d949a036a8a6c2fc5/pkg/dns/azure/dns.go#L112
https://github.com/openshift/cluster-ingress-operator/blob/e412fd75a7dae32a47665dfcaa4aa82940e2f751/pkg/dns/alibaba/dns.go#L119